### ISSUE TITLE
Add a "Silence media audio while the game window is in the background" Option

### ISSUE THREAD

**Issue by WinterPhoenix** [Facepunch/garrysmod-requests#1594]
So it started with this commit: https://commits.facepunch.com/361399

That broke an existing use-case in my servers where players are given the option to control Media Audio, specifically coming from CEF and BASS, and whether or not it's muted when the game isn't focused.
![image](https://user-images.githubusercontent.com/2799393/81510197-1622dd00-92de-11ea-96d0-00b97fb6393f.png)

This was a useful feature since it allowed playing music and other media coming from GMod, while it's in the background, while also not hearing footsteps or any other game sounds.

So on that note, **I'd like to request adding a Boolean ConVar + Setting in the Audio Options to allow players to control this functionality.**

We already have `snd_musicvolume` which controls "Music" separately from Game Audio, so there's existing precedence for not having audio controlled so flatly.

As part of this, **I would also recommend that CEF audio be brought under the control during focus changes,** similar to how BASS was just with the added functionality of this proposed option.

**Comment by Hazbelll** [comment]
I agree, this would be beneficial for finer-grain control of audio sources for both clients and servers to take advantage of, such as OP's own example.
